lib/MarathonEventBusClient.js

```diff
@@ -101,7 +108,9 @@ MarathonEventBusClient.prototype.subscribe = function () {
         url = self.options.marathonProtocol + "://" + self.options.marathonHost + ":" + self.options.marathonPort+self.options.marathonUri;
 
     // Allow unauthorized HTTPS requests
-    let eventSourceInitDict = { rejectUnauthorized: false };
+    let eventSourceInitDict = self.options.marathonHeaders;
```

Comment:
Not sure if this is a good idea here... First, the headers need to be in a `headers` property (See https://www.npmjs.com/package/eventsource#setting-http-request-headers), which I don't see in the code (usage example missing?). 

Furthermore, I'd like it better if you would have a whitelist of allowed header keys, and check if the `marathonHeaders` contains valid headers, and only using the ones which pass the whitelist.